test/helpers.js

```diff
@@ -0,0 +1,186 @@
+      var privKey = testHelpers.rings[i].getPrivateKey();
+
+      addr[name] = {};
+      addr[name].pubKey = pubKey;
+      addr[name].privKey = privKey;
+      addr[name].base58 = bcoin.primitives.Address.fromHash(bcoin.crypto.hash160(pubKey)).toString();
+    }
+
+    return addr;
+  },
+
+  getNode: async () => {
+    var node = new TrustIsRisk.FullNode({network: "regtest", passphrase: "secret"});
+
+    await node.open();
+    await node.connect();
+    node.startSync();
+
+    return node;
+  },
+
+  getWalletDB: async (node) => {
+    var walletDB = new WalletDB({
+      network: "regtest",
+      db: "memory",
+      client: new bcoin.node.NodeClient(node)
+    });
+
+    await walletDB.open();
+    await walletDB.connect();
+
+    return walletDB;
+  },
+
+  getWallet: async (walletDB, id) => {
+    var options = {
+      id,
+      passphrase: "secret",
+      witness: false,
+      type: "pubkeyhash"
+    };
+
+    return walletDB.create(options); 
+  },
+
+  mineBlock: async (node, rewardAddress) => {
+    var block = await node.miner.mineBlock(node.chain.tip, rewardAddress);
+    await node.chain.add(block);
+    return node.getBlock(node.chain.tip.hash);
+  },
+
+  time: async (milliseconds) => {
+    return new Promise((resolve, reject) => {
+      setTimeout(resolve, milliseconds);
+    });
+  },
+
+  bufferToScript: (data) => {
+    return `0x${Number(data.length).toString(16)} 0x${data.toString("hex")}`;
+  },
+
+  getP2PKHOutput: (dest, value) => {
+    var address = bcoin.primitives.Address.fromBase58(dest);
+    var script = bcoin.script.fromPubkeyhash(address.hash);
+    
+    return new bcoin.primitives.Output({script, value});
+  },
+
+  getP2PKHInput: (pubKey, prevout) => {
+    if (!prevout) {
+      prevout = { // Don't care
+        hash: "v0pnhphaf4r5wz63j60vnh27s1bftl260qq621y458tn0g4x64u64yqz6d7qi6i8",
+        index: 2
+      };
+    }
+
+    return new bcoin.primitives.Input({
+      prevout,
+      script: bcoin.script.fromString(
+          // Don't care about the signature
+          "0x47 0x3044022035e32834c6ee4db1696cc06762feca2809d865ca12a3b98c801f3f451341a2570220573bf3ffef55f2651e1563acc0a22f8056222f277f5ddf17dd583d4edd40fa6001 "
+          + testHelpers.bufferToScript(pubKey))
+    }); 
+  },
+
+  getOneOfTwoMultisigOutput: (originPubKey, destPubKey, value) => {
+    return new bcoin.primitives.Output({
+      script: bcoin.script.fromMultisig(1, 2, [originPubKey, destPubKey]),
+      value
+    });
+  },
+
+  getTrustIncreasingMTX: (originPubKey, destPubKey, value) => {
+    return new bcoin.primitives.MTX({
+      inputs: [
+        testHelpers.getP2PKHInput(originPubKey)
+      ],
+      outputs: [
+        testHelpers.getOneOfTwoMultisigOutput(originPubKey, destPubKey, value)
+      ]
+    });
+  },
+
+  applyGraph: (tir, fileName, addr) => {
+    var graph = require(fileName);
+
+    for (var origin in graph) {
+      var neighbours = graph[origin];
+      for (var dest in neighbours) {
```

Comment:
`of` requires an iterator, and objects apparently don't have one. What is wrong with `in`?